### DISCUSSION TITLE
support for make dsn for oracle

### DISCUSSION THREAD

**Discussion by darshan92kumar** [sfu-db/connector-x#294] [Q&A]
Hi,
is kereberised DB connection supported ? I feel there should be support for makedsn which should be passed to read_sql as a param in case of oracle db. Passing plain string as URI does not work all the time

**Comment by wangxiaoying** [comment]
Hi @darshan92kumar , thanks for the suggestion. Right now we only support using connection string. Underlying we use [rust oracle driver](https://docs.rs/oracle/0.0.3/oracle/index.html) to connect to oracle, and the read_sql interface is only a python binding. So we can only support the connection configurations supported by [this driver](https://docs.rs/oracle/latest/oracle/struct.Connection.html). Can you give an example of when the string URI cannot work?
